tests/screenshot/apps/components/VideoPlayer.js

```diff
@@ -0,0 +1,74 @@
+import VideoPlayer from '../../../../VideoPlayer';
+import {MediaControls} from '../../../../MediaPlayer';
+
+const posterUrl = 'http://media.w3.org/2010/05/sintel/poster.png';
```

Comment:
When importing images from media servers, there is a high probability of a timing failure.
It seems better to change to getting the video locally.

e.g) import posterUrl from '../../images/poster.png';